src/AsyncStatsD.ts

```diff
@@ -35,29 +35,29 @@ export class AsyncStatsD {
     increment = async (name: string, value: number = 1, tags?: Tags) => {
         try {
             await this.wrapStatsdPromise(this._increment(name, value, tags));
-        } catch (err) {
+        } catch (err: any) {
```

Comment:
set [useUnknownInCatchVariables](https://www.typescriptlang.org/tsconfig#useUnknownInCatchVariables) to `false` in tsconfig

Reply:
Done. Thanks! @gioragutt